File: pascal/control_flow/if_statement/if_statement_formatting.pas
program IfStatementFormatting;

begin
  if (1 > 2) then
    writeln('1 > 2');

  if (1 > 2) then writeln('1 > 2');

  if (1 > 2) then
  begin
    writeln('1 > 2');
  end;

  if (2 > 1) then
    writeln('2 > 1');

  if (2 > 1) then writeln('2 > 1');

  if (2 > 1) then
  begin
    writeln('2 > 1');
  end;
end.

File: java/control_flow/if_statement/IfStatementFormatting.java
class IfStatementFormatting {
  public static void main(String[] args) {
    boolean condition;

    condition = (1 > 2);

    if (condition)
      System.out.println("1 > 2");

    condition = (2 > 1);

    if (condition)
      System.out.println("2 > 1");

    System.out.println();

    if (1 > 2)
      System.out.println("1 > 2");

    if (2 > 1)
      System.out.println("2 > 1");

    System.out.println();

    if (1 > 2) System.out.println("1 > 2");

    if (2 > 1) System.out.println("2 > 1");

    System.out.println();

    if (1 > 2) {
      System.out.println("1 > 2");
    }

    if (2 > 1) {
      System.out.println("2 > 1");
    }

    System.out.println();
  }
}
